### PR TITLE
Workbench changes

### PR DESCRIPTION
**Pull request by jcolomb**

### CODE REVIEW

**Comment by jcolomb** [comment]
or should we add another way to make comments ?

**Comment by tobyhodges** [comment]
Here's a nice example of a README being updated after transition to the Workbench: https://github.com/datacarpentry/r-raster-vector-geospatial/pull/382

File: index.md [hunk 1]
```diff
@@ -10,7 +10,6 @@ Through lessons learned in this training, researchers will enhance their underst
 
 <p>Funding Acknowledgement: The first iteration of this project was funded by <a href="https://www.turing.ac.uk">The Alan Turing Institute</a> - <a href="https://www.turing.ac.uk/research/asg">AI for Science and Government</a> (ASG) Research Programme from October 2021 to March 2022.</p>
 
-{% comment %} This is a comment in Liquid {% endcomment %}
```
Comment (tobyhodges) [inline index.md:13]: You could substitute in `<!-- HTML comments -->`? Or, @zkamvar,  is there a specific comment syntax for (R)Markdown files?

Comment (zkamvar) [reply index.md:13]: HTML comments is correct!

**Comment by jcolomb** [comment]
> 

both there and here, the best would be to have an updated contributing.md file, is there any in the template ?

**Comment by zkamvar** [comment]
> > Here's a nice example of a README being updated after transition to the Workbench: [datacarpentry/r-raster-vector-geospatial#382](https://github.com/datacarpentry/r-raster-vector-geospatial/pull/382)
> 
> both there and here, the best would be to have an updated contributing.md file, is there any in the template ?

Why yes there is! It's here: https://github.com/carpentries/sandpaper/blob/main/inst/templates/contributing-template.txt

... uhm this is embarassing. I have not actually done anything to update it from the former template -_-. I'm currently battling a particularly nasty bug (https://github.com/carpentries/sandpaper/issues/406) and I will do my best to update it when I get time.

**Comment by jcolomb** [comment]
ok tried to make some changes at https://github.com/carpentries/sandpaper/compare/main...jcolomb:sandpaper:patch-1

**Comment by zkamvar** [comment]
> will probably be make changes there and update here using some sandpaper function, right?

The update will be copy/paste since this is a markdown file that {sandpaper} only touches at the creation of the lesson.

**Pull request merged**